llvm/test/CodeGen/AMDGPU/llvm.amdgcn.image.atomic.pk.add.ll

```diff
@@ -44,6 +44,14 @@ main_body:
   ret float 1.0
 }
 
+define amdgpu_ps float @atomic_pk_add_bf16_1d_v4(<8 x i32> inreg %rsrc, <4 x bfloat> %data, i32 %s) {
```

Comment:
Should also include a test with no use of the return value, plus with some cache bits set